libvmi/libvmi.h

```diff
@@ -687,13 +687,15 @@ typedef struct page_info {
 
     union {
         struct {
+            addr_t pte_value_prev;
```

Comment:
Could you add a comment as to when this field is set? Perhaps adding a short description into`notes` would help as well where you put the information you have in the PR description. Thanks!